host/frontend/host-orchestrator/static/js/index.js

```diff
@@ -89,8 +89,7 @@ window.addEventListener('load', e => {
   let selectDeviceCb = deviceId => {
     return new Promise((resolve, reject) => {
       let client = window.open(
-          `/devices/${deviceId}/files/client.html?deviceId=${deviceId}`,
-          deviceId);
+          `/devices/${deviceId}/files/client.html`, deviceId);
```

Comment:
can this line be moved up as it's shorter?

Reply:
ran clang-format on the line

Reply:
Ran cllang-format